doc/source/serve/deployment-graph-cookbook.md

```diff
@@ -0,0 +1,60 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.6
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+(serve-deployment-graph-cookbook)=
+
+# Deployment Graph Cookbook
+
+### Chain Nodes
+
+The example shows how to chain nodes using the same class and having different args passed in
+
++++
+
+```python
+import ray
+from ray import serve
+from ray.experimental.dag.input_node import InputNode
+from ray.serve.drivers import DAGDriver
```

Comment:
I think it's probably better to have a snippet at top of this doc as "imports" section so all the subsequent cells can be much shorter in length. We want to to the exact opposite of current e2e documentation

Reply:
I will do this in further pr if we have lots of duplicates import (i think we will have :) ).